libsolidity/inlineasm/AsmScopeFiller.cpp

```diff
@@ -47,11 +47,9 @@ bool ScopeFiller::operator()(Label const& _item)
 	if (!m_currentScope->registerLabel(_item.name))
 	{
 		//@TODO secondary location
-		m_errors.push_back(make_shared<Error>(
-			Error::Type::DeclarationError,
-			"Label name " + _item.name + " already taken in this scope.",
-			_item.location
-		));
+		m_errorReporter.declarationError(_item.location,
```

Comment:
This applies to all: please start every parameter on a new line if it has to be broken up to multiple lines.